xds/src/test/java/io/grpc/xds/RingHashLoadBalancerTest.java

```diff
@@ -1112,6 +1075,17 @@ public void nameResolutionErrorWithActiveSubchannels() {
     verifyNoMoreInteractions(helper);
   }
 
+  @Test
+  public void duplicateAddresses() {
+    RingHashConfig config = new RingHashConfig(10, 100);
+    List<EquivalentAddressGroup> servers =
+        createRepeatedServerAddrs(1, 2, 3);
```

Comment:
nit

```suggestion
    List<EquivalentAddressGroup> servers = createRepeatedServerAddrs(1, 2, 3);
```

Reply:
Why is it better to split the arguments between lines rather than split at the '=' ?

Reply:
My point was the split wasn't needed - the whole line fits into the line length: https://github.com/grpc/grpc-java/blob/3c5c2be7125d57ca48d69ad6aa2682e6d4094487/buildscripts/checkstyle.xml#L48-L49